index.html

```diff
@@ -0,0 +1,311 @@
+          </div>
+        </div>
+      </div>
+      <div class="row flex_wrap_reverse">
+        <div class="col_text">
+          <div class="col_header">
+            <i class="fas fa-hand-point-up fa-lg"></i> Beautiful and interactive
+          </div>
+          <div>
+            Static is out, dynamic is in.
+            The contemporary look and interactive features people have come to love and expect from modern webpages, now applied to scholarly articles:
+            tooltips, public annotations, table of contents, figure viewer, and more.
+          </div>
+        </div>
+        <div class="col_image"></div>
+      </div>
+    </section>
+    <section>
+      <div class="row flex_wrap">
+        <div class="col_image"></div>
+        <div class="col_text">
+          <div class="col_header">
+            <i class="fas fa-lock-open fa-lg"></i> Free and open-source
+          </div>
+          <div>
+            There's no need to pay to produce brilliant manuscripts.
+            Manubot is free and open-source (aside from workflow integration with <a href="http://github.com/">GitHub</a> and <a href="https://travis-ci.org/">TravisCI</a>, which can be substituted if desired).
+            Its tools are developed and supported by a community of responsive and passionate software developers and researchers.
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="special_section">
+      <div class="section_header">
+        Made With Manubot
+      </div>
+      <div class="row flex_wrap">
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+      </div>
+      <div class="row flex_wrap">
+        <a href="" class="button_link">
+          <i class="fas fa-book-open fa-lg"></i> Full Catalog
+        </a>
+      </div>
+    </section>
+    <section>
+      <div class="section_header">
+        Our Philosophy
+      </div>
+      <div class="row">
+        <div>
+          Manuscripts are the primary basis of scholarly communication, yet the process of creating them has not kept in step with the advancement of technology or the evolving best-practices of science.<br />
+          <br />
+          Traditionally, authoring of manuscripts takes place in a closed manner.
+          While most scientific material is eventually distributed to some kind of public medium, the process of developing and reviewing the material is usually done in private, contained to a handful of individuals.
+          The tools used are often outmoded, restrictive, and overall poorly-suited for the task of collaborative, scientific writing.
```

Comment:
This phrasing may beg the question "what's wrong with Google Docs"?  Do we want to draw more language from the meta review to more precisely contrast Manubot with existing collaborative writing tools?

Reply:
I tried to capture this with "poorly suited to *scientific* writing", which I think is where google docs fails.

Overall, I think we need to think of the landing page as a giant billboard/poster for Manubot, just key bullet points, very high level language, and let other pages go into more detail. I think it's fine to be more vague here. We can have an FAQ page in the docs that we can link to on the landing page that can include "Why not just use Google Docs?"